src/pages/home/ReportScreen.js

```diff
@@ -207,11 +214,17 @@ class ReportScreen extends React.Component {
         const reportID = getReportID(this.props.route);
         const addWorkspaceRoomPendingAction = lodashGet(this.props.report, 'pendingFields.addWorkspaceRoom');
         const addWorkspaceRoomErrors = lodashGet(this.props.report, 'errorFields.addWorkspaceRoom');
-        const isReportLoaded = this.props.report && this.props.report.reportID === reportID;
-        const isTransitioning = !isReportLoaded;
         return (
             <Freeze
                 freeze={this.props.isSmallScreenWidth && this.props.isDrawerOpen}
+                placeholder={(
+                    <ScreenWrapper
+                        style={[styles.appContent, styles.flex1, {marginTop: this.state.viewportOffsetTop}]}
```

Comment:
Flex1 is a duplicate here – `BaseScreenWrapper` already has that style: https://github.com/Expensify/App/blob/2518e21a658c3ca30868350dbb16b6c0de33bff7/src/components/ScreenWrapper/BaseScreenWrapper.js#L76

```suggestion
                        style={[styles.appContent, {marginTop: this.state.viewportOffsetTop}]}
```